Proton/Sources/Swift/Core/LayoutManager.swift

```diff
@@ -56,6 +56,17 @@ class LayoutManager: NSLayoutManager {
         return layoutManagerDelegate?.paragraphStyle ?? NSParagraphStyle()
     }
 
+    public func setNeedsDisplay(forAttachment attachment: Attachment)
```

Comment:
What is the exact use case when an attachment doesn't belong to the storage but system asks for its layout? Assuming this is a performance improvement and it definitely doesn't hurt, just trying to understand the exact scenario.
Nitpick: styling of braces is not very consistent in these two methods.

Reply:
at the moment, these functions are not used. I have been thinking about something, but that may still rely only on `setNeedsLayout`. Typically, this function would be used for updating typesetting changes. 

I'll fix the styling.